shared-helpers/src/locales/general.json

```diff
@@ -10,10 +10,18 @@
   "account.createAccount": "Create Account",
   "account.errorFetchingApplications": "Error fetching applications",
   "account.haveAnAccount": "Already have an account?",
+  "account.allMyApplications": "All my applications",
```

Comment:
Do you mind alphabetizing these? W lines selected, I use command + shift + p --> sort lines ascending in VSCode

Reply:
Learn something new everyday

Reply:
One call out here, since this whole file was alphabetized we will have to be careful when pull this over to the jurisdictions. There is a chance that we might not catch if one of the moved keys is a different translation (this has happened in the past). And I'd also recommend that the alphabetizing is also run on those jurisdiction's repos so that any keys that are unique to that repo are also placed in the right place.